core/src/value/convert/from.rs

```diff
@@ -16,6 +16,76 @@ use either::{Either, Left, Right};
 #[cfg(feature = "indexmap")]
 use indexmap::{IndexMap, IndexSet};
 
+#[cfg(feature = "chrono")]
+impl<'js> FromJs<'js> for chrono::DateTime<chrono::Utc> {
+    fn from_js(ctx: Ctx<'js>, value: Value<'js>) -> Result<chrono::DateTime<chrono::Utc>> {
+        use chrono::TimeZone;
+
+        let global = unsafe { crate::qjs::JS_GetGlobalObject(ctx.ctx) };
+        let date_constructor = unsafe {
+            crate::qjs::JS_GetPropertyStr(
```

Comment:
Use [`Object::get`](https://docs.rs/rquickjs/latest/rquickjs/struct.Object.html#method.get) method to access property by name.